packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -0,0 +1,812 @@
+            }
+        }
+
+        &__poi-confirm-example {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+
+            &--generic-error-msg {
+                .dc-hint-box {
+                    justify-content: flex-start;
+                    width: 100% !important;
+
+                    &__icon {
+                        min-width: fit-content;
+                        align-self: baseline;
+                    }
+                }
+            }
+
+            .account-form__fieldset {
+                max-width: unset;
+            }
+
+            &--status-message {
+                margin-top: 1.6rem;
+                text-align: center;
+                width: 100%;
+            }
+
+            &_container {
+                border: 1px solid var(--general-active);
+                border-radius: 0.8rem;
+
+                @include desktop() {
+                    padding: 1.6rem;
+                }
+
+                @include mobile() {
+                    margin-bottom: 7rem;
+                    padding: 1.6rem 1.6rem 0;
+                }
+
+                .account__scrollbars_container {
+                    padding: 0;
+
+                    &--grid-layout {
+                        height: auto !important;
+
+                        @include mobile() {
+                            padding: 0 1.6rem;
+                        }
+                    }
+
+                    .account-form__section {
+                        align-items: center;
+                        justify-content: space-between;
+                        margin-top: 0.8rem;
+
+                        @include mobile() {
+                            margin-bottom: 0;
+                        }
+                    }
+                }
+            }
+
+            &_wrapper {
+                transition: transform 0.35s ease, opacity 0.35s linear;
+                transform-origin: top;
+                opacity: 1;
+                padding: 1.6rem;
+                max-width: 67rem;
+
+                @include mobile() {
+                    padding-inline: 0;
+                }
+
+                &--exit {
+                    transform: scale(1, 0);
+                    opacity: 0;
+                }
+            }
+
+            .da-inline-note-with-icon {
+                margin-top: 0;
+                padding: 0.8rem;
+                display: flex;
+                background: var(--status-alert-background);
+                border-radius: 0.8rem;
+                gap: 0.8rem;
+                line-height: 1.4rem;
+            }
+
+            .account__scrollbars_container--grid-layout {
+                grid-gap: 0;
+            }
+
+            .account-form__section-side-note {
+                @include mobile {
+                    justify-content: center;
+                    display: flex;
+                }
+            }
+
+            .account-form__section-content {
```

Comment:
@yauheni-deriv everything starting with .account can be nested under `.account {`, then under `&-form {`, and so on :)